CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ Unreleased
 * Added a new method `validateElement` to both do a screenshot comparison and test accessibility
 * Log information on what selenium version is being used for the test run.
 
+### Fixed
+* Fixed the issue with Aggregate Translations CLI default options overriding the configuration file options.
+* Fixed an issue with the 'exclude' option in Aggregate Translations setup function.
```

Comment:
Can you add a short description about what issue was fixed for the exclude option?

Reply:
I'd recommend adding a link to the GitHub issue like so after the description.
```suggestion
* Fixed an issue with the 'exclude' option in Aggregate Translations setup function. [#264](https://github.com/cerner/terra-toolkit/issues/264)
```